mcs/class/Facades/System.Drawing.Common/monodroid_System.Drawing.Common.dll.sources

```diff
@@ -0,0 +1,15 @@
+#include System.Drawing.Common.dll.sources
+../../../../external/corefx/src/System.Drawing.Primitives/src/System/Drawing/Color.cs
+../../../../external/corefx/src/Common/src/System/Drawing/KnownColor.cs
+../../../../external/corefx/src/System.Drawing.Primitives/src/System/Drawing/Point.cs
+../../../../external/corefx/src/System.Drawing.Primitives/src/System/Drawing/PointF.cs
+../../../../external/corefx/src/System.Drawing.Primitives/src/System/Drawing/Rectangle.cs
+../../../../external/corefx/src/System.Drawing.Primitives/src/System/Drawing/RectangleF.cs
+../../../../external/corefx/src/System.Drawing.Primitives/src/System/Drawing/Size.cs
+../../../../external/corefx/src/System.Drawing.Primitives/src/System/Drawing/SizeF.cs
+../../../../external/corefx/src/Common/src/System/Numerics/Hashing/HashHelpers.cs
+../../../../external/corefx/src/Common/src/System/Drawing/KnownColorTable.cs
+../../../../external/corefx/src/Common/src/System/Drawing/ColorTable.cs
+../../../../external/corefx/src/Common/src/System/Drawing/ColorUtil.netcoreapp21.cs
+../../../build/common/SR.cs
```

Comment:
There must be a better way to include `mcs/build/common/SR.cs`, maybe it's auto-included if this weren't a facade?

Reply:
that's what we do but not sure we actually need it here

Reply:
`SR.Format` is used by `Color.cs`

Reply:
ok